vue/sbc-common-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sbc-common-components",
-  "version": "2.4.48",
+  "version": "2.4.49",
```

Comment:
I think we can go with 2.5.0 or 3.0.0 , since its a major change, and package-lock have lot of change